keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -305,9 +310,29 @@ In order to update the PVC ownerReference, the `buildControllerRoles` will be up
 
 ### Test Plan
 
-1. Unit tests
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+#### Unit tests
+
+In `pkg/controller/statefulset/`,
```

Comment:
Please update following the template from https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md?plain=1#L257-L328, ie. here you should have:
```
<package>: <date> - <current test coverage>
```

Reply:
done